### ISSUE TITLE
xorg-xrandr doesn't have the xrandr command

### ISSUE THREAD

**Issue by WMCB-Tech** [termux/x11-packages#271]
**Problem description**
Recently installed `xorg-xrandr` after Installation, it doesn't have `xrandr` command included (for somehow idk what's missing when creating a PR about `xorg-xrandr` package)

**Steps to reproduce**
1. Install `x11-repo` and `xorg-xrandr`

2. Query Files with `dpkg-query -L`, showing no `xrandr` program (only the docs)

![Screenshot_2021-03-29-21-04-41-17.jpg](https://user-images.githubusercontent.com/65075766/112840872-69f30c00-90d2-11eb-9452-dc77a5e57cfe.jpg)

**Expected behavior**
It should have `xrandr` executable to my knowledge

so i created this package within PR, and the CI logs says build success, but i was surprised that there's no `xrandr` executable

**System information**
```
Application version:
0.107
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://grimler.se/termux-packages-24/ stable main
# game-repo (sources.list.d/game.list)
deb https://grimler.se/game-packages-24 games stable
# science-repo (sources.list.d/science.list)
deb https://grimler.se/science-packages-24 science stable
# unstable-repo (sources.list.d/unstable.list)
deb https://ipfs.io/ipns/k51qzi5uqu5dj05z8mr958kwvrg7a0wqouj5nnoo5uqu1btnsljvpznfaav9nk unstable main
# x11-repo (sources.list.d/x11.list)
deb https://ipfs.io/ipns/k51qzi5uqu5dgu3homski160l4t4bmp52vb6dbgxb5bda90rewnwg64wnkwxj4 x11 main
Updatable packages:
All packages up to date
Android version:
10
Kernel build information:
Linux localhost 4.14.117-perf+ #2 SMP PREEMPT Thu Mar 11 23:38:07 CST 2021 aarch64 Android
Device manufacturer:
OPPO
Device model:
CPH1933
```

**Comment by Grimler91** [comment]
Really need to fix build-package check so that it reports error if package only contains license..

**Comment by xeffyr** [comment]
@WMCB-Tech Issue is that you haven't looked into sources and determined the required build steps. Here should be `./autogen.sh` in pre-configure.

***

build-package.sh indeed should verify that it doesn't create empty or license-only package.

**Comment by xeffyr** [comment]
Should be fixed in package version `1.5.1-1`.
